data.sql

```diff
@@ -1,6 +1,43 @@
 /* Populate database with sample data. */
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Charmander', '08-02-2020', 0, false, -11),
+              ('Plantmon', '15-11-2022', 2, true, -5.7),
+              ('Squirtle', '2-04-1993', 3, false, -12.13),
+              ('Angemon', '12-06-2005', 1, true, -45),
+              ('Boarmon', '07-06-2005', 7, true, 20.4),
+              ('Blossom', '13-10-1998', 3, true, 17),
+              ('Agumon', '03-02-2020', 0, true, 10.23),
+              ('Gabumon', '15-11-2018', 2, true, 8),
+              ('Pikachu', '07-01-2021', 1, false, 15.04),
+              ('Devimon', '12-04-2017', 5, true, 11);
 
-INSERT INTO animals VALUES (1, 'Agumon', '02-03-2020', 0, true, 10.23);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '15-11-2018', 2, true, 8);
-INSERT INTO animals VALUES (3, 'Pikachu', '07-01-2021', 1, false, 15.04);
-INSERT INTO animals VALUES (4, 'Devimon', '12-05-2017', 5, true, 11);
\ No newline at end of file
+INSERT INTO owners
+              (full_name, age)
+            VALUES
+              ('Sam Smith', 34), 
+              ('Jennifer Orwell', 19), 
+              ('Bob', 45), 
+              ('Melody Pond', 77), 
+              ('Dean Winchester', 14),
+              ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
+BEGIN TRANSACTION;
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
```

Comment:
- [ ] If an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead, kindly let us update our animals using names instead.